quackosm/osm_extracts/movisda.py

```
@@ -0,0 +1,136 @@
+    The download URL is built from the `prefix` property of each feature (already containing the
+    trailing dash, e.g. `AD-`) by appending `latest.osm.pbf`.
+
+    Args:
+        geojson_url (str): URL of the GeoJSON index file.
+        pbf_base_url (str): Base URL for the PBF files.
+        source_enum (OsmExtractSource): Source enum value used for ids and parent.
+
+    Returns:
+        list[OpenStreetMapExtract]: List of parsed extracts.
+    """
+    result = requests.get(geojson_url, headers={"User-Agent": _USER_AGENT})
```

Ruff (0.15.15)

[error] Probable use of `requests` call without timeout

(S113)